app/models/manageiq/providers/ansible_tower/automation_manager.rb

```diff
@@ -28,6 +28,7 @@ class ManageIQ::Providers::AnsibleTower::AutomationManager < ManageIQ::Providers
   require_nested :RefreshWorker
 
   has_many :configuration_workflows, :dependent => :destroy, :foreign_key => "manager_id", :inverse_of => :manager
+  has_many :job_templates, :class_name => "ConfigurationScript", :dependent => :destroy, :foreign_key => "manager_id", :inverse_of => :manager
```

Comment:
@Fryguy It ends up, we'll need something here for the graph refresh to work properly.  That will be used in the persister app/models/manageiq/providers/ansible_tower/inventory/persister/automation_manager.rb (in this same PR)

Reply:
right, so the query used in InventoryCollection can collide with others, otherwise they'll start to delete each other's records. (it's the same for old refresh in this sense)

Reply:
@jameswnl  can you print what the SQL queries look like, for ```ems.job_templates``` and ```ems.configuration_workflows```